Source/WebCore/css/html.css

```diff
@@ -1406,8 +1405,11 @@ iframe {
     border: 2px inset;
 }
 
-bdi, output {
-    unicode-bidi: isolate;
+address, blockquote, center, div, figure, figcaption, footer, form, header, hr,
+legend, listing, main, p, plaintext, pre, summary, xmp, article, aside, h1, h2,
+h3, h4, h5, h6, hgroup, nav, section, search, table, caption, colgroup, col,
+thead, tbody, tfoot, tr, td, th, dir, dd, dl, dt, menu, ol, ul, li, bdi, output {
```

Comment:
How do we know that this is the set that should have this style attached, and none others? Is that documented somewhere?

Reply:
It is from web specification here - https://html.spec.whatwg.org/multipage/rendering.html#bidi-rendering

Reply:
Oh, excellent. I would add that in your ChangeLog to indicate where that came from.

Reply:
It is already in commit. :-)